packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -70,5 +70,52 @@ void main() {
     expect(find.text('B'), findsOneWidget);
     box = tester.renderObject(find.byType(ExpansionPanelList));
     expect(box.size.height - oldHeight, greaterThanOrEqualTo(100.0)); // 100 + some margin
+
+    // multiple panel list test
+    await tester.pumpWidget(
+      new MaterialApp(
+        home: new ListView(
+          children: <ExpansionPanelList>[
+            new ExpansionPanelList(
+              expansionCallback: (int _index, bool _isExpanded){
+                index = _index;
+                isExpanded = _isExpanded;
```

Comment:
since you don't actually interact with this widget, you can probably just leave the callback empty (or maybe omit it altogether?)